### PR TITLE
Support Braze Epic ophanComponentId parameter

### PR DESCRIPTION
**Pull request by rhystmills**
## What does this change?
The Braze Epic now required Marketing to specify an `ophanComponentId`, which is used as a specific campaign identifier in our logs to Ophan.

## Why?
To allow Marketing to analyse the performance of specific variants of the Braze Epic.

[Related PR for @guardian/braze-components](https://github.com/guardian/braze-components/pull/142).

### CODE REVIEW

File: src/web/lib/braze/parseBrazeEpicParams.test.ts [hunk 1]
```diff
@@ -127,6 +136,22 @@ describe('parseBrazeEpicParams', () => {
...
+			buttonText: 'Button',
+			ophanComponentId: 'epic_123',
+		};
+
+		const got = parseBrazeEpicParams(dataFromBraze);
+
+		expect(got).toBeNull();
+	});
+
+	it('returns null when the trackingId is missing', () => {
```
Comment (adamnfish) [inline src/web/lib/braze/parseBrazeEpicParams.test.ts:154]: It looks like this test has two missing fields (`buttonUrl` and `ophanComponentId`). Should this be two separate tests?

**Review by adamnfish — APPROVED** [review]
Question about the tests, but looks good otherwise.

**Review by liywjl — APPROVED** [review]
LGTM 👍

File: src/web/lib/braze/parseBrazeEpicParams.test.ts (hunk 1 above)
Comment (rhystmills) [reply src/web/lib/braze/parseBrazeEpicParams.test.ts]: Good spot, sorted now.

**Pull request merged**